Makefile

```diff
@@ -166,25 +140,10 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	  FEED_REVISION=`cd $$FEED_DIR; $(REVISION)`; \
 	  echo "Feed $$FEED: repository from $$FEED_GIT_REPO, git branch \"$$FEED_GIT_BRANCH_ESC\", revision $$FEED_REVISION" >> $$VERSION_FILE; \
 	done
-	# copy different firmwares (like vpn, minimal) including imagebuilder
-	for DIR_ABS in $(IB_BUILD_DIR)/imgbldr/bin/*; do \
-	  TARGET_DIR=$(FW_TARGET_DIR)/$$(basename $$DIR_ABS); \
-	  rm -rf $$TARGET_DIR; \
-	  mv $$DIR_ABS $$TARGET_DIR; \
-	  cp $(FW_TARGET_DIR)/$$VERSION_FILE $$TARGET_DIR/; \
-	  for FILE in $$TARGET_DIR/openwrt*; do \
-	    [ -e "$$FILE" ] || continue; \
-	    NEWNAME="$${FILE/openwrt-/kathleen-}"; \
-	    NEWNAME="$${NEWNAME/ar71xx-generic-/}"; \
-	    NEWNAME="$${NEWNAME/mpc85xx-generic-/}"; \
-	    NEWNAME="$${NEWNAME/squashfs-/}"; \
-	    mv "$$FILE" "$$NEWNAME"; \
-	  done; \
-	done;
+	./assemble_firmware.sh -p "$(PROFILES)" -i $(IB_FILE) -t $(FW_TARGET_DIR) -u "$(PACKAGES_LIST_DEFAULT)"
```

Comment:
@bittorf mentions this upstream-thing in https://github.com/weimarnetz/weimarnetz/issues/216#issuecomment-164053487
Maybe he can tell more regarding upstream, even this might still be only relevant for OpenWrt and not discussed with the LEDE-guys